final/chromosome.py

```diff
@@ -0,0 +1,14 @@
+class Chromosome: 
+    def __init__(self, genes):
+        self.genes = genes
+        self.fitness = 0
+
+    def swap(self, first, second):
+        self.genes[first], self.genes[second] = self.genes[second], self.genes[first]
+    
+    def calc_fitness(self):
```

Comment:
we can use @property and change the name of method to `fitness`(see @property in python). for speedup of code we can use `if self.fitness is not 0: return self.fitness`.

Reply:
Did you notice that fitness is one of our class properties?
We need a method to calculate fitness.
Could you explain more?

Reply:
I think @property is just a decorator instead of using getter and setter.